opencog/nlp/chatbot-psi/contexts.scm

```diff
@@ -3,6 +3,9 @@
 (load "states.scm")
 (load "utils.scm")
 
+; For var-decl etc
+(load "../relex2logic/rule-utils.scm")
+
```

Comment:
Attempt to deal with that in https://github.com/opencog/opencog/pull/2583, would be nice if @linas can have a look at that PR as well